src/index.html

```diff
@@ -25,8 +25,6 @@ <h3>Today's To Do.</h3>
       </div>
       <div class="display"></div>
     </div>
-    <!-- <div class="display"></div>
```

Comment:
- [ ] Great job 👏🏻 However I couldn't open your app in the browser even after running `npm install`, `npm run build`, 'npm run start`, or `npm run dev`. instead, I get this error message 👇🏻 as shown in the screenshot. Kindly Make sure to fix this error 😉 👍🏻 

- My computer:

![Screenshot from 2022-10-06 13-14-15](https://user-images.githubusercontent.com/79058364/194299062-a2111eca-2240-4426-bdce-62dccc22e491.png)

- Second Computer
<img width="1488" alt="image" src="https://user-images.githubusercontent.com/79058364/194298806-c71e3b08-5bdd-4f1e-bf95-38bc27894322.png">